avocado/plugins/spawners/podman.py

```diff
@@ -103,12 +112,20 @@ async def spawn_task(self, runtime_task):
             runtime_task.status = msg
             return False
 
+        if mount_status_server_socket:
+            status_server_opts = (
+                "--privileged",
+                "-v", "%s:%s" % (status_server_uri, mounted_status_server_socket)
+            )
+        else:
+            status_server_opts = ("--net=host", )
```

Comment:
dumb question: Is this disabling network on the container? Or just using a different method? What about if the test needs internet connection?

Reply:
> dumb question: Is this disabling network on the container? Or just using a different method? What about if the test needs internet connection?

No, it just reverts back to the `slirp4netns` default.  For networking to be completely disabled, `--net=none` would have to be provided.